packages/orbit-components/src/CountryFlag/index.js

```diff
@@ -76,12 +76,12 @@ StyledShadow.defaultProps = {
 };
 
 function getCountryProps(code: ?string, name: ?string): {| code: string, name: ?string |} {
```

Comment:
the line was not changed, but  I guess it should be: `getCountryProps(code?: string, name?: string): {| code: string, name: string |}`, right?

Reply:
Returned `name` can still be `null`, but I changed the rest.